### PR TITLE
spark-publish: publish participant_address

### PR DESCRIPTION
**Pull request by bajtos**
Rename the measurement field `wallet_address` to `participant_address`, as discussed in https://github.com/filecoin-station/spark-evaluate/pull/10

### CODE REVIEW

File: spark-publish/index.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ export const publish = async ({
       id,
       spark_version,
       zinnia_version,
-      wallet_address,
+      wallet_address as participant_address,
```
Comment (juliangruber) [inline spark-publish/index.js:19]: can we also reflect this change in the schema?

**Review by juliangruber — CHANGES_REQUESTED** [review]

File: spark-publish/index.js (hunk 1 above)
Comment (juliangruber) [reply spark-publish/index.js:19]: and in the data the spark client submits in the first place?

Comment (bajtos) [reply spark-publish/index.js:19]: Great ideas 👍🏻

**Comment by bajtos** [comment]
@juliangruber updated the DB schema and REST API. I also added few lines of code to support existing SPARK nodes. PTAL

**Comment by bajtos** [comment]
SPARK module change: https://github.com/filecoin-station/spark/pull/26

**Review by juliangruber — APPROVED** [review]

**Pull request merged**